registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ApiName.java

```diff
@@ -197,6 +197,7 @@ public enum ApiName {
 	IDREPOUPDATEDRAFT,
 	IDREPOPUBLISHDRAFT,
 	IDREPOEXTRACTBIOMETRICS,
+	CRUPMANAGERAPI,

```

Comment:
Proper url for "CRUPMANAGERAPI" keyword should be defined in mosip-config under registration-processor-default.properties file.